readme.md

```diff
@@ -0,0 +1,149 @@
+
+
+![Book Store App.jpeg](readmeImg%2FBook%20Store%20App.jpeg)
+# Book Store App
+## 🔬 Project Overview
+
+This project is a Book Store application built using Java and Spring Boot. It is designed to manage a catalog of books, provide user authentication and authorization, and ensure secure access to various operations. The project supports CRUD operations for book management and includes a user-friendly interface for accessing the API documentation through Swagger UI.
+
+* * *
+
+## 🛠️ Technologies and Tools
+
+- **Spring Boot**: Core framework for building the application.
+- **Spring Data JPA**: Manages database interactions.
+- **Spring Security**: Handles authentication and authorization.
+- **Liquibase**: Manages database migrations.
+- **Maven**: Project management and dependency resolution.
+- **H2 Database**: In-memory database for testing (CI).
+- **MySQL**: Main database for the application.
+- **Swagger**: API documentation and testing tool.
+- **JUnit & Mockito**: For unit testing and mocking.
+- **GitHub Actions**: Continuous Integration (CI) tool.
+- **Checkstyle**: Ensures code style consistency.
+
+* * *
+
+## 💡Features
+
+- User Registration and Authentication
+- Book Browsing and Searching
+- Shopping Cart Management
+- Order Processing
+- User Profile Management
+- Admin Dashboard for Managing Books and Orders
+- Secure RESTful API Endpoints
+
+* * *
+
+## 🏗️️ Installation
```

Comment:
What about the `.env` file? I think we need it at least for docker-compose, but it can also be useful in simple boot run, to not change the `application.properties` file but use env variables instead